tests/from.bats

```diff
@@ -367,3 +367,20 @@ load helpers
   [ "$status" -eq 0 ]
   buildah rm ${cid}
 }
+
+@test "from security-opt test" {
```

Comment:
I don't think this test is necessary; that's what the last one in `selinux.bats` is supposed to do, I think. If you agree, feel free to update that test (adding `buildah inspect`); or LMK and I can work it into my big bats PR.

I would like to see added tests for `--annotations` and `--env`, possibly just piggybacked onto the existing tests in `bud.bats`, but it might be easier for me to add that using my new BATS framework once/if that gets approved.

Reply:
@edsantiago if you could massage these changes into your PR or a separate one, that would be awesome.